fastlane.gemspec

```diff
@@ -96,7 +96,7 @@ Gem::Specification.new do |spec|
 
   # The Google API Client gem is *not* API stable between minor versions - hence the specific version locking here.
   # If you upgrade this gem, make sure to upgrade the users of it as well.
-  spec.add_dependency('google-api-client', '>= 0.29.2', '< 0.37.0') # Google API Client to access Play Publishing API
+  spec.add_dependency('google-api-client', '>= 0.29.2', '< 0.38.0') # Google API Client to access Play Publishing API
```

Comment:
I wonder if we shall bump the minimal version requirement to `0.37.0`, which contains the support of `in_app_update_priority`

Reply:
Yes. That makes sense. In fact I can change the min to `0.37.0` and max to `< 0.39.0` (0.39.2 is the latest but is only 12 hours old. Whereas the release before 0.39.0 is 0.38.0 and is about one month old)

Reply:
@dlackty I have pushed the change. Thank you for the suggestion.